sdk/digitaltwins/Azure.DigitalTwins.Core/tests/SessionRecords/ComponentTests/Component_Lifecycle.json

```diff
@@ -15,7 +18,7 @@
       "ResponseHeaders": {
         "Content-Length": "212",
         "Content-Type": "application/json; charset=utf-8",
-        "Date": "Wed, 28 Oct 2020 18:17:58 GMT",
+        "Date": "Fri, 30 Oct 2020 16:55:41 GMT",
```

Comment:
Out of curiosity, what caused all these test string changes?

Reply:
Rerecording the tests